mkosi/config.py

```diff
@@ -192,6 +192,22 @@ def config_match_enum(dest: str, value: str, namespace: argparse.Namespace) -> b
 
     return config_match_enum
 
+def config_make_list_matcher(type: Type[enum.Enum], delimiter: str) -> ConfigMatchCallback:
+    def config_match_list(dest: str, value: str, namespace: argparse.Namespace) -> bool:
+        matched = False
```

Comment:
No need for this variable, just return immediately if the match function returns true once